### DISCUSSION TITLE
Can't RTSP Wyze Cam V2

### DISCUSSION THREAD

**Discussion by keplenk** [gtxaspec/wz_mini_hacks#517] [Q&A]
Hi,

I have a few WyzeCam V2s and just found out about this method of using RTSP on these cameras.

I had to downgrade my firmware to 4.9.8.1002 and I was able to successfully flash it as I can now SSH into it and access the web server.

However, I'm not sure what I'm doing wrong but I can't get an RTSP feed from it.  I use VLC to test but it just won't work.

Here is the screenshot of my RTSP config from the web

![image](https://user-images.githubusercontent.com/47620544/236101468-f31522f1-f90b-4a57-ad18-99a5381aaa16.png)

Can someone please help this newbie out with this issue?  I feel like it could be something simple.

EDIT: I also tried setting the RTSP_AUTH_DISABLE to True --> no go
         I also tried  video6_unicast, video6_unicast, unicast --> still nogo

**Comment by akivacp** [comment]
Set the double radio boxes to true as well (the ones on top and/or bottom)

**Reply by keplenk** [reply]
I didn't realize that my screenshot was old but yes I did try those too earlier

![image](https://user-images.githubusercontent.com/47620544/236112741-6f489f6c-e08b-4302-97f0-b507ee4266b1.png)

I also checked the low res options but still no go.

Should I put some information in the boxes .. (like the FPS, ENC_Parameters) .. or should I just leave those blank?

**Reply by bjs-pdx** [reply]
Re-read this section of setup:  https://github.com/gtxaspec/wz_mini_hacks#rtsp-streaming

You need the Bit Rate and ENC parameters for it to work.

Try these settings:

RTSP_HI_RES_ENABLED="true"
RTSP_HI_RES_ENABLE_AUDIO="true"
RTSP_HI_RES_MAX_BITRATE="2048"
RTSP_HI_RES_TARGET_BITRATE="1024"
RTSP_HI_RES_ENC_PARAMETER="2"
RTSP_HI_RES_FPS=""
